cluster/gce/util.sh

```diff
@@ -53,6 +53,17 @@ function join_csv {
 
 # Verify prereqs
 function verify-prereqs {
+  if [[ "${ENABLE_EXPERIMENTAL_API}" == "true" ]]; then
```

Comment:
I think appending is a bed idea. RUNTIME_CONFIG defined by user may contain "experimental/v1=false", and if we appended "experimental/v1=true" we would have wrong configuration. My solution only checks if RUNTIME_CONFIG is consistent with ENABLE_EXPERIMENTAL_API, what is much safer.